### PR TITLE
Fix macOS installation stage in CMakeLists.txt

### PR DESCRIPTION
**Pull request by Marcool04**
Hi there!
I know, as per your clear answers in this: https://github.com/CDrummond/cantata/issues/1671, this: https://github.com/CDrummond/cantata/issues/1543 and this: https://github.com/CDrummond/cantata/issues/1534 issue for example, that you no longer support macOS.
I also noted from the README.md that Cantata is now in a bug-fix only state, and that any improvements would have to come from PR's.
This is why I wanted to share this small patch I had to make to the CMakeLists.txt file in order to get the software to build on macOS.
I also wish to take this chance to thank you very much for this very useful and pleasant piece of software which I use on a daily basis. Great work!
Best regards,
Mark.

### CODE REVIEW

**Comment by Marcool04** [comment]
I also added a commit that updates the installation instructions for mac, mostly revolving around the fact that qt homebrew formula is not qt6 and so we actually need to specify qt5 and point to it in cmake stage.

File: README [hunk 1]
```diff
@@ -790,39 +790,17 @@ These steps assume the following structure:
...
 
-2. brew install cmake taglib ffmpeg openssl qt
+2. brew install cmake taglib ffmpeg openssl qt5
 
 3. Create the icon
 
     cd src/mac
     ./createicon.sh
 
-From here you have a couple of options:
```
Comment (CDrummond) [inline README:800]: Why have you removed this section?

File: README [hunk 2]
```diff
@@ -790,39 +790,17 @@ These steps assume the following structure:
...
+4. Build the software and create the Cantata.app mac app bundle
 
     cd build
-    cmake ../src -DCMAKE_PREFIX_PATH=/usr/local/opt/qt/ -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=$(pwd)/../install
+    export Qt5_DIR="/usr/local/opt/qt5/lib/cmake"
+    cmake -S "../src" -B "./" -DCMAKE_INSTALL_PREFIX="../install" -DENABLE_TAGLIB=ON -DCMAKE_PREFIX_PATH="/usr/local/opt/qt5/;/usr/local/opt/qt5/lib/cmake/Qt5Widgets/" -DCMAKE_BUILD_TYPE=Release
     make
     make install
-
-Create Installer
```
Comment (CDrummond) [inline README:823]: Why is this removed?

File: README (hunk 1 above)
Comment (Marcool04) [reply README:800]: Because I tried to get it to build in QtCreator but couldn't… Honestly, that is probably due to my lack of experience with the software. If you feel it's best to include it none the less I'm fine with that.

File: README (hunk 2 above)
Comment (Marcool04) [reply README:823]: My mistake. Fixed in 466e9a6784007ddaefd349b40a5b095fa9e25b8f

File: README (hunk 1 above)
Comment (CDrummond) [reply README:800]: I'd rather you left it in there, removing jsut because makes no sense to me.

Comment (Marcool04) [reply README:800]: Sure your choice of course. But as it is written currently it doesn't build…

Comment (Marcool04) [reply README:800]: Fixed in b7c355e2c29cf9358d1382bf8f7889c50ffb6b3e

**Comment by CDrummond** [comment]
Thanks!

**Comment by Marcool04** [comment]
You're welcome, thank you for the fast merge 👍

**Comment by dphoyes** [comment]
Hi @Marcool04 - Please could you take a look at #1711 and check that it works for you? It reverts one of your commits that I believe introduces a bug. Thanks!

**Comment by Marcool04** [comment]
Hi @dphoyes , sorry if I somehow messed up… just testing now (the build is rather slow on my aging 2012 macbook pro…)
